less/current/base/variables.less

```diff
@@ -33,6 +33,24 @@
 @color-medium-blue: #08c;
 @color-dark-blue: #005580;
 
+// Color variants
```

Comment:
a11y work in the following weeks might need any of these so leaving them as comment, will clean up after finishing